packages/app/src/files-manager/components/files-manager-panel/files-manager-panel.styles.tsx

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
The name of this file should be `styled`, not `styles` like another files with style 😉

Reply:
https://github.com/mudita/mudita-center/pull/905/commits/f5ec35d0de138b5fc78080461cc5e4b24599e745

Reply:
Good, but remove additional space before `styled` 🙈